GeckoView Nightly((20190805-140939) manual update required

https://tools.taskcluster.net/groups/NAroY_B_R4OKq3tlvYf1mA/tasks/KoJ_3vPaTR6K9NUftJam2A/details


```
> Task :lib-crash:compileDebugKotlin FAILED
e: /build/android-components/components/lib/crash/src/main/java/mozilla/components/lib/crash/service/MozillaSocorroService.kt: (42, 28): None of the following functions can be called with the arguments supplied: 
@AnyThread @NonNull public open fun sendCrashReport(@NonNull p0: Context, @NonNull p1: Intent, @NonNull p2: String): GeckoResult<String!> defined in org.mozilla.geckoview.CrashReporter
@AnyThread @NonNull public open fun sendCrashReport(@NonNull p0: Context, @NonNull p1: Bundle, @NonNull p2: String): GeckoResult<String!> defined in org.mozilla.geckoview.CrashReporter
@AnyThread @NonNull public open fun sendCrashReport(@NonNull p0: Context, @NonNull p1: File, @NonNull p2: File, @NonNull p3: String): GeckoResult<String!> defined in org.mozilla.geckoview.CrashReporter
@AnyThread @NonNull public open fun sendCrashReport(@NonNull p0: Context, @NonNull p1: File, @NonNull p2: (Mutable)Map<String!, String!>, @NonNull p3: String): GeckoResult<String!> defined in org.mozilla.geckoview.CrashReporter

FAILURE: Build failed with an exception
```.

--- a/buildSrc/src/main/java/Gecko.kt
+++ b/buildSrc/src/main/java/Gecko.kt
@@ -6,7 +6,7 @@ internal object GeckoVersions {
     /**
      * GeckoView Nightly Version.
      */
-    const val nightly_version = "70.0.20190802094835"
+    const val nightly_version = "70.0.20190805095413"
 
     /**
      * GeckoView Beta Version.

--- a/components/lib/crash/src/main/java/mozilla/components/lib/crash/service/MozillaSocorroService.kt
+++ b/components/lib/crash/src/main/java/mozilla/components/lib/crash/service/MozillaSocorroService.kt
@@ -43,7 +43,6 @@ class MozillaSocorroService(
             applicationContext,
             File(crash.minidumpPath),
             File(crash.extrasPath),
-            crash.minidumpSuccess,
             appName)
     }
 }

